test/junit/scala/collection/ToConserveTest.scala

```diff
@@ -0,0 +1,90 @@
+package scala.collection
+
+import org.junit.Assert.{assertNotSame, assertSame}
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+import scala.collection.{immutable => i, mutable => m}
+import scala.language.implicitConversions
+import scala.{collection => c}
+
+@RunWith(classOf[JUnit4])
```

Comment:
probably don't need the `RunWith`

Reply:
IntelliJ doesn't run the tests for me without it...

Reply:
huh, weird :(